lib/phoenix/sync/controller.ex

```diff
@@ -43,35 +43,34 @@ defmodule Phoenix.Sync.Controller do
 
   ## Shape definitions
 
-  Shape definitions can be any of the following:
+  See `Phoenix.Sync.shape!/2` for examples of shape definitions
 
-  - An `Ecto.Schema` module:
+  ## Interruptible calls
```

Comment:
Does this make sense without additional context? I.e.: could we add a small section to the main docs explains what interruptibility is and when/why it's useful? Then link to that here to provide context?